### ISSUE TITLE
Reproducibility concern.

### ISSUE THREAD

**Issue by guangpintao** [rulixiang/wetr#14]
I repeated this code several times, but it didn't reach 62.6% (61 around  W.O. reg & CRF). Btw, why can CRF be increased by 2.2, while I use single-scale CRF with an increase of less than 1.0? At the same time, the loss of reg did not bring a 1.2 performance improvement (not small), and the result I got was about 0.5.

**Comment by guangpintao** [comment]
The performance I reproduced 63.0 is lower  than  AA&LR ACM MM’2021 (63.9 VOC2012 valid announced in paper).

**Comment by rulixiang** [comment]
For experiments with `reg loss`, the log and weight files can be found at https://github.com/rulixiang/afa/issues/3#issuecomment-1107694274. The full test code is now available [here](https://github.com/rulixiang/afa/blob/master/scripts/test_msc_flip.py). I did use multiscale and flip inference before CRF. They're common post-processing steps for semantic segmentation, so I didn't expatriate the details in paper. 
Since I didn't save the weights without `reg loss`, I repeated the experiments and got close results again. The new log and weights of the reproduced results are available here ([log & weights](https://drive.google.com/drive/folders/1B83gbWBnqHoOEn7RKRhQv5gpxJ7Rg-W2?usp=sharing)).
I don't what's wrong with your experiments, but it looks like the reproduction is fine.
